__tests__/app.test.js

```diff
@@ -116,3 +115,58 @@ describe('get articles',()=>{
   });
 
 })
+
+describe('GET /api/articles/:article_id/comments',()=>{
+  test("return 200 and comments by article's id", () => {
+    return request(app)
+      .get("/api/articles/5/comments")
+      .expect(200)
+      .then(({body})=>{
+        expect(body.comments).toHaveLength(2);
+            body.comments.forEach((comment) => {
+              expect(comment).toHaveProperty("comment_id", expect.any(Number)),
+              expect(comment).toHaveProperty("body", expect.any(String)),
+              expect(comment).toHaveProperty("votes", expect.any(Number)),
+              expect(comment).toHaveProperty("author", expect.any(String)),
+              expect(comment).toHaveProperty("created_at", expect.any(String)),
+              expect(comment).toHaveProperty("article_id", expect.any(Number))
+            })
+      })
+  });
```

Comment:
We can be explicit here about what the article id is as we want to make sure we only have comments from this article.